src/containers/permissions/assign-permissions-to-data-set/AssignPermissionsToDataSetModalBody.js

```diff
@@ -71,8 +72,9 @@ const AssignPermissionsToDataSetModalBody = ({
     .join(', ');
 
   const confirmText = assignPermissionsToAllProperties
-    ? `Please confirm you want to assign ${permissions} to "${targetDataSetTitle}" and all its columns.`
-    : `Please confirm you want to assign ${permissions} to "${targetDataSetTitle}".`;
+    ? `Please confirm you want to assign ${permissions} to "${targetDataSetTitles.join(', ')}" `
+      + 'datasets and all of their columns.'
+    : `Please confirm you want to assign ${permissions} to "${targetDataSetTitles.join(', ')}" datasets.`;
```

Comment:
Another thought here ... I think it might be useful to specifically call out that the columns will not get permissions.